src/coreclr/vm/jithelpers.cpp

```diff
@@ -5736,6 +5736,65 @@ HCIMPL3(void, JIT_VTableProfile64, Object* obj, CORINFO_METHOD_HANDLE baseMethod
 }
 HCIMPLEND
 
+HCIMPL1(void, JIT_CountProfile32, volatile LONG* pCounter)
+{
+    FCALL_CONTRACT;
+    FC_GC_POLL_NOT_NEEDED();
+
+    LONG count = *pCounter;
+    LONG delta = 1;
+
+    if (count > 0)
+    {
+        unsigned int logCount = 0;
+        BitScanReverse(&logCount, count);
+
+        if (logCount >= 13)
```

Comment:
Can you have a comment about `13`, probably just point to the paper `Scalable Statistics Counters" by Dice, Lev, and Moir (SPAA’13).` at least?

Reply:
That paper won't help you understand why this value is 13, but the design note I added will.